spec/requests/v0/health_care_applications_spec.rb

```diff
@@ -455,7 +455,7 @@ def self.expect_async_submit
       context 'when hca service raises an error' do
         before do
           test_veteran.delete('email')
-          allow_any_instance_of(HCA::Service).to receive(:post) do
+          allow_any_instance_of(HCA::Service).to receive(:submit_form) do
```

Comment:
This allow block was causing tests to fail. Can someone explain why we were using `post` as the receive method for the `HCA:Service` instead of simply using `submit_form`? I'm genuinely curious. In this instance, is there a benefit to using an inherited method as opposed to a method native to the class?